oshi-core/src/main/java/oshi/util/GlobalConfig.java

```diff
@@ -0,0 +1,166 @@
+/**
+ * OSHI (https://github.com/oshi/oshi)
+ *
+ * Copyright (c) 2010 - 2019 The OSHI Project Team:
+ * https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The global configuration utility. See
+ * {@code src/main/resources/default.properties} for default values.
+ * 
+ * @author cilki
+ * @since 4.0.0
+ */
+public final class GlobalConfig {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GlobalConfig.class);
+
+    private static final Properties configuration = new Properties();
+
+    static {
+        try (InputStream in = GlobalConfig.class.getResourceAsStream("/default.properties")) {
+            if (in != null)
```

Comment:
General comment throughout -- my IDE will reformat all conditionals using curly brackets.   I won't hold up merging for this, but will probably follow up with a "reformat" commit to standardize the code style.

Reply:
Are you using a custom formatting profile? I'm just using the "Java Conventions" profile in Eclipse.

Reply:
Huh.  So, actually it doesn't fix it.  Maybe I got tired of seeing CheckStyle warnings on it or something.  Or it was drilled into me in some programming class somewhere that it's optional in the language but a good idea (except when playing code golf.)

Reply:
I was just wondering if there's a style guide or a special xml formatting profile that I should be using for OSHI. Some projects keep the profile in the repo if they are using custom conventions ([for example](https://github.com/spring-projects/spring-batch/blob/master/spring-eclipse-code-conventions.xml)).

This is something that we should fix because I can't run my formatter on most existing source files without it changing a bunch of things.

Reply:
I uploaded my Eclipse format file here:
https://github.com/oshi/oshi/tree/master/oshi-dist/src/main/resources/eclipse
It's the "Sun Java Conventions" with one alteration:  spaces instead of tabs.

Reply:
And possibly 120 character line width.

Reply:
Exactly what I was looking for. Thanks.